src/components/dialog/ImportModelTemplateDialog.vue

```diff
@@ -83,7 +83,8 @@ onUnmounted(() => {
 <style scoped lang="scss">
 .carousel-img {
   cursor: zoom-in;
-  height: 100px;
```

Comment:
keep default size

Reply:
If I keep the default size, the carousel image is too small for detailed images:
![image](https://user-images.githubusercontent.com/20184007/220594599-aa299e29-4d6b-483d-8c5e-fd26c0165e33.png)

The goal of this commit it to enlarge it.
Should I use `min-height` here too?